styleguide/js/zoolander.js

```diff
@@ -1,3 +1,26 @@
+jQuery(document).ready(function(){
```

Comment:
I mentioned to DP I wrote this section sort of long form just to be super separate about the js until it was approved, made sure it was functioning properly and good to go. If its all working as everyone expects I can slim this section down pretty significantly.